scripting/get5.sp

```diff
@@ -417,6 +426,8 @@ public void OnPluginStart() {
   g_OnSeriesInit = CreateGlobalForward("Get5_OnSeriesInit", ET_Ignore);
   g_OnSeriesResult =
       CreateGlobalForward("Get5_OnSeriesResult", ET_Ignore, Param_Cell, Param_Cell, Param_Cell);
+  g_OnSeriesResultWithoutWinners =
```

Comment:
> So why check if was a forfeit if it wasn't? Same idea for match stats, why try to get the stats if there's no stats for that match?

I don't understand the problem. If you're trying to optimize the code, the cost of the extra forward and code maintenance massively outweighs any trivial performance change. 

I would still use the existing forward, and pass MatchTeam_TeamNone as the winner.